apps/back-office/src/app/(dashboard)/clients/[id]/diagnosisAndTreatmentPlan/components/TreatmentPlanTemplate.tsx

```
@@ -11,43 +12,438 @@ import {
+    }));
+  };
+
+  // Get current survey data from the model
+  const getCurrentSurveyData = () => {
+    console.log("=== DEBUG: Getting current survey data ===");
+    console.log("surveyModelRef.current exists:", !!surveyModelRef.current);
+    console.log("surveyModelRef.current?.data:", surveyModelRef.current?.data);
+    console.log("loadedSurveyData:", loadedSurveyData);
+    console.log("surveyAnswers state:", surveyAnswers);
+    
+    if (surveyModelRef.current && surveyModelRef.current.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)